pilot/prompts/development/implement_changes.prompt

```diff
@@ -11,6 +11,6 @@ the full contents of the updated file, without skipping over any content
 ------------------------end_of_format---------------------------
 
 **IMPORTANT**: Your reply MUST NOT omit any code in the new implementation or substitute anything with comments like `// .. rest of the code goes here ..` or `# insert existing code here`, because I will overwrite the existing file with the content you provide. Output ONLY the content for this file, without additional explanation, suggestions or notes. Your output MUST start with ``` and MUST end with ``` and include only the complete file contents.
-**IMPORTANT**: If the user must configure something manually, mark the line that needs user configuration with `INPUT_REQUIRED {input_description}` comment,  where `input_description` is a description of what needs to be added here by the user. Use appropriate syntax for comments in the file you're saving (for example `// INPUT_REQUIRED {input_description}` in JavaScript). If the file type doesn't support comments (eg JSON), don't add any.
+**IMPORTANT**: Avoid hardcoding configuration values (eg. port, database url or server secret string) in code. Instead, use appropriate settings file (for example `.env`) for all user-configurable settings. Mark line(s) that need user configuration with `INPUT_REQUIRED {input_description}` comment,  where `input_description` is a description of what needs to be added in that line by the user. Use appropriate syntax for comments in the file you're saving (for example `# INPUT_REQUIRED {input_description}` in `.env`). If the file type doesn't support comments (eg JSON), don't add any.
```

Comment:
It is the hope of this author that this heartfelt plea will induce the LLM to be more considerate with respect to peppering `INPUT_REQUIRED` throughout the code and the resulting mental anguish that causes to the user.